### ISSUE TITLE
Failed to execute installation Apple Script

### ISSUE THREAD

**Issue by corintho** [alex35mil/NeoHub#4]
## What happened?
I tried to install it on Sonoma.
I gave it administrator access, but it failed with the following error
I had to manually create /usr/local/lib
After I created the folder, it worked as expected

## Error details
```
Failed to execute installation Apple Script
App: Version 0.1.1 (Build 7057133)
macOS: Version 14.0 (Build 23A344)
Arch: arm64
Context: InstallationView#body
Metadata:
    ["NSAppleScriptErrorAppName": NeoHub, "NSAppleScriptErrorRange": NSRange: {0, 240}, "NSAppleScriptErrorBriefMessage": cp: /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: symlink: Versions/Current/NeoHubLib: No such file or directory
cp: symlink: Versions/Current/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/NeoHubLib: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/_CodeSignature: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature/CodeResources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/Resources: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources/Info.plist: No such file or directory
cp: symlink: A: No such file or directory, "NSAppleScriptErrorNumber": 1, "NSAppleScriptErrorMessage": cp: /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework: No such file or directory
cp: symlink: Versions/Current/NeoHubLib: No such file or directory
cp: symlink: Versions/Current/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/NeoHubLib: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/_CodeSignature: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/_CodeSignature/CodeResources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /Applications/NeoHub.app/Contents/Frameworks/NeoHubLib.framework/Versions/A/Resources: unable to copy extended attributes to /usr/local/lib/NeoHubLib.framework/Versions/A/Resources: No such file or directory
cp: /usr/local/lib/NeoHubLib.framework/Versions/A/Resources/Info.plist: No such file or directory
cp: symlink: A: No such file or directory]
```

**Comment by alex35mil** [comment]
Ah, right. I'll fix it. Thanks for the report!

**Comment by idelice** [comment]
I'm experiencing the same thing. Any news on this @alex35mil ?

**Comment by falcucci** [comment]
@Delice0 I'll look at this soon :)

**Comment by falcucci** [comment]
@Delice0 @corintho I couldn't reproduce it but I force to create the parent path in case it doesnt exists, so make sure to delete only the lib path as previously and make sure to make a backup of lib's content.

I have added this on my PR https://github.com/alex35mil/NeoHub/pull/8

you can also test it out downloading it here: https://github.com/falcucci/NeoHub/releases/tag/v0.0.2

**Comment by idelice** [comment]
> https://github.com/falcucci/NeoHub/releases/tag/v0.0.2

Using that tag fixed all issues for me. Thank you!

FYI I had to change the default keymap from `CMD + CONTROL + N` to `CMD + CONTROL + M `in order to get rid of the weird toggling-between-all-applications issue i've mentioned in neovide repo, and it also fixed the issue where the toggle view disappearing right away when using the shortcut. Now I can actually interact with the toggle view lol

**Comment by falcucci** [comment]
@Delice0 thank you, I confirm that the switcher shortcut was conflicting with the latest neovide instance shortcut.

I've updated the release at my fork, just download again and it would be fixed.

**Comment by idelice** [comment]
> @Delice0 thank you, I confirm that the switcher shortcut was conflicting with the latest neovide instance shortcut.
> 
> 
> 
> I've updated the release at my fork, just download again and it would be fixed.

It says that the latest release is from 9 days ago?

**Comment by falcucci** [comment]
yep that one, just install it again

**Comment by alex35mil** [comment]
It should be fixed in [`0.2.1`](https://github.com/alex35mil/NeoHub/releases/tag/v0.2.1`), but please see #15.

**Comment by alex35mil** [comment]
FYI I just updated `neovim` to the latest stable and `neovide` to the latest master (`0.13.1`) and things work again with `neohub@0.2.1`.
